exporter.go

```diff
@@ -92,7 +92,7 @@ func NewCephExporter(conn *rados.Conn, cluster string, config string, withRGW bo
 
 	if withRGW {
 		c.collectors = append(c.collectors,
-			collectors.NewRGWCollector(cluster, config),
+			collectors.NewRGWCollector(cluster, config, true),
```

Comment:
Can we expose the background collection flag through command line options? In case someone chooses to use it (or not use it) they wouldn't need to make a code change.